lucille-core/src/test/java/com/kmwllc/lucille/indexer/OpenSearchIndexerTest.java

```diff
@@ -73,6 +80,13 @@ private void setupOpenSearchClient() throws IOException {
     DeleteByQueryResponse mockDeleteByQueryResponse = Mockito.mock(DeleteByQueryResponse.class);
     when(mockClient.deleteByQuery(any(DeleteByQueryRequest.class))).thenReturn(mockDeleteByQueryResponse);
     when(mockDeleteByQueryResponse.failures()).thenReturn(new ArrayList<>());
+
+    mockFailPingClient = Mockito.mock(OpenSearchClient.class);
```

Comment:
Since we're only using this in one test -- testOpenSearchIndexerException() -- we can set up the mock in that test method. It doesn't need to be instance variable of OpenSearchIndexerTest that's initialized in setup() before each test method -- that would make more sense if this resource was used in most or all of the tests.